machine_learning/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>machine_learning</name>
```

Comment:
I think that just like with sub and boat, we should establish a  mil namespace. I propose that all packages in this repo start with `mil_`. What say you?